src/python/deepgnn/graph_engine/snark/meta_merger.py

```diff
@@ -48,97 +49,80 @@ def __init__(self, fs: fsspec.AbstractFileSystem, path: str) -> None:
         self._from_file()
 
     def _from_file(self):
```

Comment:
you don't need this function anymore, right? Every JSON file has exactly the same schema and can be merged on the fly.